src/app/modules/products/components/featured-product-card/featured-product-card.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="featured-product-card">
+    <div class="featured-product-card-image-container">
+        <img class="featured-product-card-image" src="https://images.unsplash.com/photo-1475855581690-80accde3ae2b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80" alt="">
+    </div>
+    <div class="featured-product-card-body">
+        <div class="featured-product-card-body-header">
+            <h2 class="featured-product-card-body-title">{{product.name}}</h2>
+            <span class="featured-product-card-body-highlight">{{product.collection}}</span>
+        </div>
+        <p class="featured-product-card-body-description" *ngFor="let description of product.displayedDescription">{{description}}</p>
+        <div class="featured-product-card-body-price-container">
+            <div class="featured-product-card-body-price">{{product.price}}</div>
+            <button class="featured-product-card-body-get-notified" (click)="onClick()">Get notified</button>
+        </div>
+    </div>
```

Comment:
I removed the bottom part just because we only want to show a teaser of the product and limit the space it takes